docs/guides/custom-redirects.mdx

```diff
@@ -219,15 +231,103 @@ export default function Home() {
     <div>
       <h1>Welcome!</h1>
 
-      <SignInButton signInFallbackRedirectUrl="/dashboard" signUpFallbackRedirectUrl="/onboarding">
+      <SignInButton fallbackRedirectUrl="/dashboard" signUpFallbackRedirectUrl="/onboarding">
         Sign in
       </SignInButton>
 
-      <SignUpButton signInFallbackRedirectUrl="/dashboard" signUpFallbackRedirectUrl="/onboarding">
+      <SignUpButton signInFallbackRedirectUrl="/dashboard" fallbackRedirectUrl="/onboarding">
         Sign up
       </SignUpButton>
     </div>
   );
 }
 ```
 </CodeBlockTabs>
+
+### UI Components
+
+[`<SignIn>`](/docs/components/authentication/sign-in) and [`<SignUp>`](/docs/components/authentication/sign-up) also take these properties.
+
+<Callout type="info">
+It is recommended to define both `signInFallbackRedirectUrl` and `signUpFallbackRedirectUrl` redirects in each component as some users may choose to sign up instead after attempting to sign in.
+</Callout>
+
+<Tabs type="framework" items={["Next.js", "React", "Remix", "Gatsby"]}>
```

Comment:
If the examples aren't changing per framework, then we don't need to have each framework. We can simply write "The following example is written for Next.js App Router, but is supported by any React meta framework, such as Remix or Gatsby."